### ISSUE TITLE
Add support for data that is not JSON

### ISSUE THREAD

**Issue by tkoop** [zackurben/alphavantage#282]
The LISTING_STATUS function returns a list in CSV format.  It would be nice to be able to do that with this library.

https://www.alphavantage.co/documentation/#listing-status

**Comment by dawadam** [comment]
Error is : `invalid json response body at https://www.alphavantage.co/query?apikey=xxx&function=LISTING_STATUS reason: Unexpected token s in JSON at position 0`

**Comment by dawadam** [comment]
It's possible to make conversion on the fly using library like [https://github.com/Keyang/node-csvtojson](https://github.com/Keyang/node-csvtojson).

**Comment by zackurben** [comment]
Sorry for the delay, my notifications got turned off for github.... I just added basic support for this by using the additional param `datatype=csv`

```
// example if not supported already
alpha.experimental('ENDPOINT_WITH_CSV_RETURN', { symbol: 'XYZ', datatype: 'csv'});
```

This was just published with `alphavantage@2.4.1`
